USAGEGUIDE.md

```diff
@@ -0,0 +1,291 @@
+<img alt="" role="img" src="https://blog.1password.com/posts/2021/secrets-automation-launch/header.svg"/>
+<div align="center">
+  <h1>Usage Guide</h1>
+</div>
+<h2>Table of Contents</h2>
```

Comment:
This comment applies to all occurrences of the following HTML tags: `<h2>`, `<h3>`, `<h4>`.
Please use the markdown header style instead of the HTML ones, specifically:
- `<h2>` → `##`
- `<h3>` → `###`
- `<h4>` → `####`

Using HTML tags instead of markdown header styling also breaks links to parts of this page.